### ISSUE TITLE
Missing aria-label on the node in @xyflow/svelte

### ISSUE THREAD

**Issue by GuillaumeMMM** [xyflow/xyflow#5799]
### What platform were you using when you found the bug?

- React Flow / Svelte Flow version: @xyflow/svelte 1.5.2
- Browser and version: Brave 1.88.136
- OS and version: MacOS Tahoe 26.3.1

### Live code example

https://stackblitz.com/edit/svelte-flow-ts-scmwbxma?file=src%2FFlow%2Findex.svelte,src%2FApp.svelte

### Describe the Bug

Hello there!
I'm using @xyflow/svelte, and when I add the `ariaLabel` to the node object, the `aria-label` attribute is not added to the "svelte-flow__node" HTML element. 

Adding `ariaLabel` does work as expected when using @xyflow/react. 

It looks like the `aria-label` attribute is just not passed to the `div` in `NodeWrapper.svelte`.

Thanks!

### Steps to reproduce the bug or issue

1. Create a `node: Node` using @xyflow/svelte
2. Add an `ariaLabel` to the node object
3. Look at the corresponding node in the DOM with class "svelte-flow__node"
4. The `aria-label` attribute is missing

### Expected behavior

The `aria-label` is expected to be on the HTML element.
